README.md

```diff
@@ -2,4 +2,6 @@
 
 Simple TGA file format loader written in C++.
 
-Should you have any questions don't hesitate to write me an email to raydelto@raycasters.com
+This is meant to be a library for anybody that would like to use functions to change a tga image.
+
+Should you have any questions don't hesitate to write me an email to custo2720@gmail.com
```

Comment:
Since this is being merged to the original repo, please don't erase my email address.